src/foam/u2/crunch/CrunchController.js

```diff
@@ -111,13 +112,13 @@ foam.CLASS({
             return {
               caps: capAndSections[0],
               wizCaps: capAndSections[1]
-                .filter((wizardSection) =>
-                  wizardSection.ucj === null ||
-                  (
-                    ! wizardSection.ucj.status === this.CapabilityJunctionStatus.GRANTED &&
-                    ! wizardSection.ucj.status === this.CapabilityJunctionStatus.PENDING
-                  )
+              .filter((wizardSection) =>
+                wizardSection.ucj === null ||
+                (
+                  ! foam.util.equals(wizardSection.ucj.status, this.CapabilityJunctionStatus.GRANTED ) &&
```

Comment:
Enums can be compared with ==.

Reply:
So not suppose to have made this change - sorry